packages/webapp/client/sass/application.scss

```diff
@@ -3,7 +3,7 @@ $govuk-assets-path: '/public/';
 $govuk-images-path: '#{$govuk-assets-path}images/';
 $govuk-fonts-path: '#{$govuk-assets-path}fonts/';
 $govuk-global-styles: true;
-$govuk-font-family: "Arial", sans-serif !important;
+$govuk-font-family: "GDS Transport", arial, sans-serif !important;
```

Comment:
Remove "GDS Transport", we can't use it for licensing reasons. Correct font throughout should be arial